scvi/core/modules/scanvae.py

```diff
@@ -176,8 +176,8 @@ def classify(self, x, batch_index=None):
             w_y = self.classifier(z)
         return w_y
 
-    def get_latents(self, x, y=None):
-        zs = super().get_latents(x)
+    def get_latents(self, x, y=None, batch_index=None):
```

Comment:
These functions are not actually used anywhere, we could also consider removing them now.

Reply:
We use them, but they can be replaced ofc.

Reply:
are you guys able to use instead the API as shown in the colab notebooks I linked in the PR?